querybook/webapp/components/DataDocQueryExecutions/DataDocQueryExecutions.tsx

```diff
@@ -24,136 +24,155 @@ interface IProps {
     isQueryCollapsed: boolean;
 
     changeCellContext?: (context: string) => void;
+    onChangeFromAI?: (query: string, run: boolean) => void;
```

Comment:
why don't we just use changeCellContext but also add an optional parameter to run?